lambdas/layer/app.py

```diff
@@ -0,0 +1,15 @@
+from azul import (
+    config,
+)
+from azul.chalice import AzulChaliceApp
+
+# This whole file only exists so that we can use Chalice to create the layer
+# package and is removed from the final result.
+
+app = AzulChaliceApp(app_name=config.indexer_name,
+                     unit_test=globals().get('unit_test', False))
+
+
+@app.route('foo')
+def foo():
+    return {'bar': 'foo'}
```

Comment:
```suggestion
    return {}
```

Does this even have to return anything?

Reply:
No. I changed it to just `pass`.